doc/source/symbol-tables.rst

```diff
@@ -25,17 +25,17 @@ as long as the symbol files stay in the same location.
 Windows symbol tables
 ---------------------
 
-For Windows systems, Volatility accepts a string made up of the GUID and Age of the required PDB file.  It then
+For Windows systems, Volatility accepts a string made up of the GUID and age of the required PDB file.  It then
 searches all files under the configured symbol directories under the windows subdirectory.  Any that contain metadata
-which matches the pdb name and GUID/age (or any compressed variant) will be used.  If such a symbol table cannot be found, then
+which matches the PDB name and GUID/age (or any compressed variant) will be used.  If such a symbol table cannot be found, then
 the associated PDB file will be downloaded from Microsoft's Symbol Server and converted into the appropriate JSON
 format, and will be saved in the correct location.
 
 Windows symbol tables can be manually constructed from an appropriate PDB file.  The primary tool for doing this
 is built into Volatility 3, called :file:`pdbconv.py`.  It can be run from the top-level Volatility path, using the
 following command:
 
-:command:`PYTHONPATH="." python volatility3/framework/symbols/windows/pdbconv.py`
+:command:`PYTHONPATH="."; python volatility3/framework/symbols/windows/pdbconv.py`
```

Comment:
I don't think that semi-colon belongs there.  It would run as two separate commands, rather than applying the environment variable `PYTHONPATH` to the command `python`.